### ISSUE TITLE
Implement support for multi-sensor group operation

### ISSUE THREAD

**Issue by jhodapp** [Jim-Hodapp-Coaching/ambi#11]
### Summary
Allow for multiple sensor groups (e.g. different sensor group for different rooms/contexts)

### Feature specifics

- Refactor DB schema model such that a group of sensor readings can be associated with a specific context (e.g. room name such as "Living Room" vs "Bedroom") -> This needs intentional design consideration and discussion
- Redesign dashboard to display multiple sensor groups (e.g. select a room to display)

### Not included

- This does not include updating the Edge client nor the ambi_mock_client -> this work is captured in separate issue under each respective repository
  - Note: you may want to do part of this work before or in parallel to this issue
